tests/conftest.py

```diff
@@ -1,8 +1,8 @@
 from binascii import hexlify
 from datetime import timedelta
-from ethereum import tester
+from ethereum.tools import tester as tester
 from ethereum.abi import ContractTranslator
-from ethereum.tester import ABIContract
+from tester import ABIContract
```

Comment:
This should use the full path rather than depending on a previous import:
```
from ethereum.tools.tester import ABIContract
```

The reason for this is so that it is easy for someone reading the code to quickly find provenance.  When it just says `from tester import ABIContract` the reader has to resolve the `tester` reference (which is not always a required step) before they can establish provenance.

Same comment for all other `from tester import ...` lines.